Comskip initial release

schumi2004 opened the pull request:
I fucked up with pussing commit and auto closing PR #1900
This is a new commit to introduce Comskip for the community against the current ffmpeg build (was a issue in previous attempts)

Compiled it for a few architectures without issues.

Note: Comskip is a free mpeg commercial detector. 
http://www.kaashoek.com/comskip/

ndaversa commented:
+1 lets get this in

cytec commented on cross/comskip/Makefile:
```diff
@@ -0,0 +1,24 @@
+
+PRE_CONFIGURE_TARGET = myPreConfigure
+GNU_CONFIGURE = 1
+#ADDITIONAL_LDFLAGS="-Wl,-rpath,/usr/local/ffmpeg/lib/"
```
if this is not needed remove it ;)

cytec commented on spk/comskip/Makefile:
```diff
@@ -0,0 +1,40 @@
+	install -m 755 -d $(STAGING_DIR)/var
+	install -m 644 src/comskip.ini $(STAGING_DIR)/var/comskip.ini
+	install -m 644 src/comskip.dictionary $(STAGING_DIR)/var/comskip.dictionary
+	#rm $(STAGING_DIR)/bin/ff*
```
same as above, remove the line if it's not needed

cytec commented on spk/comskip/src/installer.sh:
```diff
@@ -0,0 +1,52 @@
+
+preupgrade ()
+{
+    exit 0
```
since you copy the config file to `var` folder... when one changes those, you should backup/restore the var folder here... see: https://github.com/SynoCommunity/spksrc/blob/master/spk/syncthing/src/installer.sh#L107-L109

triksmelb commented:
+1 exciting stuff

KaraokeStu commented:
@schumi2004 If you can get @cytec 's requested changes, we can approve this PR and get it built! 👍

cytec commented on spk/comskip/Makefile:
```diff
@@ -0,0 +1,40 @@
+
+REQUIRED_DSM = 5.0
+
+DEPENDS = cross/argtable cross/ffmpeg cross/$(SPK_NAME)
```
`cross/$(SPK_NAME)` should be enough here since `cross/comskip` alreday depends on argtable and ffmpeg

schumi2004 commented:
Sorry for my late reply, this i have done:
Updated it to latest version
Fixed comments from cytec
Added digest file

I also created a wiki page for it in the past but unsure how to add it if needed:
https://github.com/SynoCommunity/spksrc/wiki/Comskip

triksmelb commented:
thanks Schumi2004

cytec commented:
LGTM \cc @KaraokeStu

maxrogers commented:
FYI, this worked for me on DSM6
